### PR TITLE
Load existing Data Imports into GAU Widget

### PR DESCRIPTION
**Pull request by dfuller114**
# Critical Changes

# Changes
When a data import row is opened from the batch gift entry table, GAU allocations will also be loaded in addition to the rest of the form.

# Issues Closed
[W-038511](https://foundation.lightning.force.com/lightning/r/agf__ADM_Work__c/a2x1E000001HqUjQAK/view)

# Community Ideas Delivered

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by dfuller114** [comment]
**lurch: attach W-038511

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000001HqUjQAK" target="_blank">W-038511</a>

File: src/lwc/geFormWidgetAllocation/geFormWidgetAllocation.js [hunk 1]
```diff
@@ -152,21 +153,59 @@ export default class GeFormWidgetAllocation extends LightningElement {
...
+    @api
+    load(data) {
+        let dataImportRow = JSON.parse(data[DI_ADDITIONAL_OBJECT.fieldApiName]).dynamicSourceByObjMappingDevName;
+        if (!dataImportRow) {
+            return;
+        }
+        let rowList = new Array();
+        let fieldMappings = GeFormService.fieldMappings;
+        let gauMappingKeys = Object.keys(fieldMappings).filter(key => {
+            return key.toLowerCase().includes('gau_allocation_1');
```
Comment (vmartinov-sforg) [inline src/lwc/geFormWidgetAllocation/geFormWidgetAllocation.js]: Can we add a constant for 'gau_allocation_1'.

**Comment by dfuller114** [comment]
@ptewson-sfdo mentioned about incorrect naming of the feature branch so created https://github.com/SalesforceFoundation/NPSP/pull/5133 to correct the format for the feature branch name.

**Pull request closed without merging**